lib/ansible/cli/doc.py

```diff
@@ -1124,24 +1114,23 @@ def add_fields(text, fields, limit, opt_indent, return_values=False, base_indent
                         conf['cli'].append(cli)
                 del opt['cli']
 
+            # add custom header for conf
             if conf:
-                text.append(DocCLI._dump_yaml({'set_via': conf}, opt_indent))
+                text.append(DocCLI._indent_lines(DocCLI._dump_yaml({'set_via': conf}), opt_indent))
 
+            # these we handle at the end of generic option processing
             version_added = opt.pop('version_added', None)
             version_added_collection = opt.pop('version_added_collection', None)
 
+            # general processing for options
             for k in sorted(opt):
                 if k.startswith('_'):
                     continue
-                if isinstance(opt[k], string_types):
-                    text.append('%s%s: %s' % (opt_indent, k,
-                                              textwrap.fill(DocCLI.tty_ify(opt[k]),
-                                                            limit - (len(k) + 2),
-                                                            subsequent_indent=opt_indent)))
-                elif isinstance(opt[k], (Sequence)) and all(isinstance(x, string_types) for x in opt[k]):
-                    text.append(DocCLI.tty_ify('%s%s: %s' % (opt_indent, k, ', '.join(opt[k]))))
+
+                if isinstance(opt[k], Sequence) and not isinstance(opt[k], string_types):
```

Comment:
Same comment as above.  I'd use `is_sequence` here.

Reply:
Fixed in a1872973319a3a785c663e2e85e9838419882af5.